bin/concurrently.spec.ts

```diff
@@ -17,14 +13,17 @@ const createKillMessage = (prefix: string) =>
  * Returns observables for its combined stdout + stderr output, close events, pid, and stdin stream.
  */
 const run = (args: string) => {
-    // TODO: This should only be transpiled once. Tests become 2.5x slower doing it in every `it`.
-    const child = spawn('ts-node --transpile-only ./concurrently.ts ' + args, {
+    // TODO: Optimally, this should only be transpiled once,
+    //       e.g. bundle in `beforeAll` and then reuse here.
+    const child = spawn(`node -r @swc-node/register ./concurrently.ts ${args}`, {
```

Comment:
Nice, I don't know much about SWC but I hope that the bin tests are much faster now?

Reply:
Yes, SWC is great for such use cases 👍 It now takes about half as long as before 🚀